planetiler-core/src/main/java/com/onthegomap/planetiler/writer/TileArchiveWriter.java

```diff
@@ -79,29 +77,19 @@ private MbtilesWriter(Iterable<FeatureGroup.TileFeatures> inputTiles, Mbtiles db
     maxTileSizesByZoom = IntStream.rangeClosed(0, config.maxzoom())
       .mapToObj(i -> new LongAccumulator(Long::max, 0))
       .toArray(LongAccumulator[]::new);
-    memoizedTiles = stats.longCounter("mbtiles_memoized_tiles");
-    featuresProcessed = stats.longCounter("mbtiles_features_processed");
+    memoizedTiles = stats.longCounter("tileset_memoized_tiles");
+    featuresProcessed = stats.longCounter("tileset_features_processed");
     Map<String, LongSupplier> countsByZoom = new LinkedHashMap<>();
     for (int zoom = config.minzoom(); zoom <= config.maxzoom(); zoom++) {
       countsByZoom.put(Integer.toString(zoom), tilesByZoom[zoom]);
     }
-    stats.counter("mbtiles_tiles_written", "zoom", () -> countsByZoom);
-  }
-
-  /** Reads all {@code features}, encodes them in parallel, and writes to {@code outputPath}. */
-  public static void writeOutput(FeatureGroup features, Path outputPath, MbtilesMetadata mbtilesMetadata,
-    PlanetilerConfig config, Stats stats) {
-    try (Mbtiles output = Mbtiles.newWriteToFileDatabase(outputPath, config.compactDb())) {
-      writeOutput(features, output, () -> FileUtils.fileSize(outputPath), mbtilesMetadata, config, stats);
-    } catch (IOException e) {
-      throw new IllegalStateException("Unable to write to " + outputPath, e);
-    }
+    stats.counter("tileset_tiles_written", "zoom", () -> countsByZoom);
   }
 
   /** Reads all {@code features}, encodes them in parallel, and writes to {@code output}. */
   public static void writeOutput(FeatureGroup features, Mbtiles output, DiskBacked fileSize,
```

Comment:
could you change `Mbtiles output` to `TileArchive output` ?

Reply:
I get a ` try-with-resources not applicable to variable type` - is this related to my JDK version?